web/web.go

```diff
@@ -55,6 +62,8 @@ func (s *Server) NewRouter() http.Handler {
 	r.HandleFunc(s.Prefix+"/Ticket/Display.html", s.ticketHandler)
 	r.HandleFunc(s.Prefix+"/Ticket/Attachment/{transactionID}/{attachmentID:[0-9]+}/{filename}", s.attachHandler)
 	r.HandleFunc(s.Prefix+"/Search/Simple.html", s.searchHandler)
+	// route to serve static content
+	r.PathPrefix(s.Prefix + "/static/").Handler(http.StripPrefix("/static/", http.FileServer(http.Dir(dir))))
```

Comment:
I think you need to strip s.Prefix + "/static"

Reply:
that works :-)